### PR TITLE
Broker error handling

### PR DESCRIPTION
**Pull request by mwodahl**
Adds exit + reboot when broker connection fails

### CODE REVIEW

File: src/acm.cpp [hunk 1]
```diff
@@ -308,8 +307,14 @@ bool ASN1_Codec::topic_available( const std::string& topic ) {
...
         }
 
+    } else if ( err == RdKafka::ERR__TRANSPORT) {
+        logger->error("cannot retrieve consumer metadata: Broker Transport Failure ");
+        logger->error("Container Exiting...will try restarting");
+        std::exit(EXIT_FAILURE);
     } else {
         logger->error("cannot retrieve consumer metadata with error: " + err2str(err));
+        logger->error("Container Exiting...will try restarting");
+        std::exit(EXIT_FAILURE);
```
Comment (dmccoystephenson) [inline src/acm.cpp:317]: The program isn't actually responsible for restarting, it's up to the environment that its deployed in to initiate the restart. Should we just say 'container exiting' here?

**Review by dmccoystephenson — APPROVED** [review]
Looks good! Just one question/suggestion regarding some error messages.

File: src/acm.cpp (hunk 1 above)
Comment (mwodahl) [reply src/acm.cpp]: Updated to just say Container Exiting

**Review by dmccoystephenson — APPROVED** [review]
Looks good!

**Review by payneBrandon — APPROVED** [review]

**Pull request merged**